Android/Fundamentals/intro-to-android/tools-needed-to-use-android.md

```diff
@@ -0,0 +1,53 @@
+# tools-needed-to-use-android
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+Android can be written on a PC, Macs, and Linux machines. To write programs or code, an IDE (Integrated Development Environment) is needed which is Android Studio. Android Studio is free to download which works on all platforms. JDK(Java Development Kit) and Android SDK is needed. JDK is used to write Java programs.
+To get started developing Android, we need Android SDK. This software development kit provides developers to build, compile, test and analyze applications. 
+The Android SDK includes:
+A mobile device emulator, which is a virtual mobile device that runs on your computer. The emulator allows you to develop and test Android applications without using a physical device. 
```

Comment:
Shorten this up and turn it into a bulleted list if possible, or as much as you can get it to be one